### ISSUE TITLE
Swap out Vision encoder?

### ISSUE THREAD

**Issue by Deltadahl** [QwenLM/Qwen2-VL#114]
Is it possible to swap out the vision encoder to a custom vision endoer (custom DINOv2 model) for these models and still run the full training script?

**Comment by logicwong** [comment]
You can try to modify the code and ckpt, but I would advise against it. We spent a lot of effort to align a vision encoder that supports dynamic resolution with the LLM, and switching it to another VE may cause a huge performance decline.

**Comment by Deltadahl** [comment]
The problem I'm having is that I need to fine tune the full model (including the VE) since I'm using it for a specific medical dataset paried with reports, but in my experiments so far, the model ignores the image completely during inference. Do you have any thoughts on how to fine tune the full model to this new domain in a good way? (currently using a dataset of 10k paired image-reports)

**Comment by logicwong** [comment]
> The problem I'm having is that I need to fine tune the full model (including the VE) since I'm using it for a specific medical dataset paried with reports, but in my experiments so far, the model ignores the image completely during inference (trained full model inc. VE for 3 epochs). Do you have any thoughts on how to fine tune the full model to this new domain in a good way? (currently using a dataset of 10k paired image-reports)

@simonJJJ Any suggestions?

**Comment by wjbmattingly** [comment]
I am struggling here too. I have 30,000 medieval manuscript pages that I am trying to fine tune on.

**Comment by Deltadahl** [comment]
Bump.
Or is there another way to do this, like weighing the loss on the vision encoder to be larger?

**Comment by HaFred** [comment]
bummer

**Comment by ronnief1** [comment]
any update on this?
